paddle/fluid/framework/ir/mkldnn/fc_mkldnn_pass.cc

```diff
@@ -32,7 +32,7 @@ class Graph;
 namespace {
 void LogEnabledOps(const int counter, const std::string& details) {
```

Comment:
This method is redefined in each pass. Some log only if count is > 0, others log all. Do you think we could extract this fuse pass logger to `mkldnn_reuse.h` and only call it?

Reply:
I saw one in cpu_quantize_pass.cc from which I copied and modified it but this is only two examples so please give me a third as per the rule of three :D
https://understandlegacycode.com/blog/refactoring-rule-of-three/#use-the-rule-of-three

Reply:
I mean we have multiple fuse pass counters with logs e.g. elt_act_mkldnn_fuse_pass.cc:102, fc_elementwise_add_mkldnn_fuse_pass.cc:119, matmul_transpose_reshape_mkldnn_fuse_pass.cc:108 etc.

I think that same code is repeated each time, with only hardcoded op name being changed. It is not particularly related to your PR, I just wanted to gather opinions if you find it good idea as future improvement.

Reply:
Ok there is some repetition even in non mkldnn passes so if we were to add such method in future we could put it in FusePassBase or some other common parent class.